lib/user.rb

```diff
@@ -0,0 +1,75 @@
+class User
+
+  @@current_username = "admin"
+  @@current_user_id = 0
```

Comment:
These should be stored in session not in model.

Reply:
Otherwise two different people connect to the server at the same time and can see each other's content.